js/ci.project.js

```diff
@@ -300,10 +300,12 @@ $(function() {
         }
         if (projectData.writePerm) {
             if(build.class === 1 && projectData.project.projectType === 1) {
-                if(projectData.release === null && projectData.preRelease === null) {
```

Comment:
I don't know about this bit, how will users know why they cant submit new releases?